### PR TITLE
Notes from hackEDU community planning meeting

### LINKED ISSUE #52: Create plan for non-labs program clubs
For the clubs that are not in the labs program:

- what will our interaction with them be?
- if they reach out for help, what will happen?

I scheduled a lunch meeting to discuss this tomorrow (Tuesday August 17th)

[cc @hellyeah, @zachlatta, @MaxWofford, @andreadeng]

### PR DESCRIPTION
**Pull request by hellyeah**
Resolves #52 #58

### CODE REVIEW

**Comment by jonleung** [comment]
@hellyeah, thoughts on keeping it the same format as our other meeting notes?

[Here is an example.](https://github.com/hackedu/hackedu/blob/community-meeting/meta/labs/meetings/15-08-20_la_clubs.md)

Sorry that this is kinda annoying, but I think it *may* have payoffs down the road though I'm not sure.

Thoughts on whether this is worth it @zachlatta and @MaxWofford?

**Comment by zachlatta** [comment]
It's valuable to include who was in attendance at meetings so it's clear who's in and not in the loop.

**Comment by hellyeah** [comment]
Should be ready to merge.

File: meta/labs/meetings/15-08-28_non_labs_clubs.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+_In attendance:_
+
+- Jonathan Leung <jonathon@hackedu.us>
+- Zach Latta <zach@hackedu.us>
+- Dave Fontenot <dave@hackedu.us>
+- Max Wofford <max@hackedu.us>
+
+-------------------------------------------------------------------------------
+- What do we want to offer non-labs teams?
```
Comment (zachlatta) [inline meta/labs/meetings/15-08-28_non_labs_clubs.md]: There should be a new line between lines 10 and 11.

**Comment by MaxWofford** [comment]
@zachlatta Is there a reason this was merged before https://github.com/hackedu/hackedu/pull/137#discussion_r38990726 was addressed?

**Comment by zachlatta** [comment]
@MaxWofford I didn't have an opportunity to leave that comment before it was merged :-/

**Comment by zachlatta** [comment]
I did create #175 that addressed my comment though.

**Pull request merged**